tests/test_formats.py

```diff
@@ -139,7 +139,7 @@ def test_entrypoint_references_with_extras():
     assert formats.python_entrypoint_reference(example) is False
 
 
-@pytest.mark.parametrize("example", ["module" "invalid-module"])
+@pytest.mark.parametrize("example", ["module", "invalid-module"])
 def test_invalid_entrypoint_references(example):
     assert formats.python_entrypoint_reference(example) is False
```

Comment:
```suggestion
    result = example == "module"
    assert formats.python_entrypoint_reference(example) is result
```

Reply:
I assume this is what was intended, @abravalheri?

Reply:
```suggestion
    result = example == "module"
    assert formats.python_entrypoint_reference(example) is result
```

Reply:
@abravalheri pre-commit will wipe this every week. Any chance you could comment? :)